www-servers/nginx-unit/nginx-unit-1.3-r1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A dynamic web and application server"
+HOMEPAGE="https://unit.nginx.org"
+SRC_URI="https://unit.nginx.org/download/unit-${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
I don't think you actually need to rename it. Plus, you can put `unit-${PV}` into `MY_P` to avoid having to repeat it twice.

Reply:
I made the MY_P change, but if it is OK with you I'd rather stick with renaming the tarball during download. "unit-x.y.tar.gz" is not very descriptive.

Reply:
Sure, it's not a big deal.

Reply:
Thanks. As far as I can tell, I complied with your other change requests. Interestingly, I only changed the Ruby support (as you can see from the diffs shown in my original comment), and the last time the ebuild was accepted by the reviewer. I guess it just goes to show that different Gentoo devs have different preferences. 😉